portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -117,6 +117,40 @@ definition {
 			locator1 = "FormFields#COLOR_SWATCH_ITEM");
 	}
 
+	macro editColorSelectFieldByRow{		
+		Type(		
+			key_fieldName = "${fieldName}",
+			key_rowNumber = "${rowNumber}",
+			locator1 = "FormFields#COLOR_FIELD_BY_ROW",
+			value1 = "${colorValue}");
+
+		FormFields.viewColorValueLabelPickerByRow(
+			fieldLabel = "${fieldLabel}",
+			fieldName = "${fieldName}",
+			rowNumber = "${rowNumber}",
+			colorValue = "${colorValue}");
+
+	}
+	
+	macro editColorSelectPickerByRow {	
+			
+		Click(
+			key_fieldLabel = "Color",
```

Comment:
@lima-rafa, shouldn't you use the variable `"${fieldLabel}"` instead of the word `Color`?

Reply:
ok